templates/components/products/carousel.html

```diff
@@ -2,31 +2,38 @@
     data-list-name="{{list}}"
     data-slick='{
         "dots": true,
-        "infinite": false,
         "mobileFirst": true,
         "slidesToShow": 2,
-        "slidesToScroll": 2,
+        "slidesToScroll": 1,
+        "slide": ".product-slide",
```

Comment:
I suggest to name it as js-'name' as well

Reply:
agree